deploy/helm/elastic-agent/templates/integrations/_system/_system_logs.tpl

```diff
@@ -0,0 +1,45 @@
+{{- define "elasticagent.system.config.logs.init" -}}
+{{- if $.Values.system.logs.enabled}}
+{{- $preset := $.Values.agent.presets.perNode -}}
+{{- $inputVal := (include "elasticagent.system.config.logs.input" $ | fromYamlArray) -}}
+{{- include "elasticagent.preset.mutate.inputs" (list $ $preset $inputVal) -}}
+{{- include "elasticagent.preset.applyOnce" (list $ $preset "elasticagent.kubernetes.pernode.preset") -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "elasticagent.system.config.logs.input" -}}
+- id: system-logs
+  type: logfile
+  use_output: {{ $.Values.system.output }}
+  data_stream:
+    namespace: {{ $.Values.system.namespace }}
+  streams:
+    - data_stream:
+        dataset: system.auth
+        type: logs
+      paths:
+        - /var/log/auth.log*
+        - /var/log/secure*
```

Comment:
Will users want to adjust these paths?

Reply:
I pushed an update that now allows the user to adjust these paths, and thank you for the inspiration, I added an example showcasing that 🙂 Feel free to unresolve if you spot anything else